smart-contracts/test/fixtures/locks.js

```diff
@@ -1,37 +1,3 @@
-const { ethers } = require('hardhat')
-const { MAX_UINT } = require('@unlock-protocol/hardhat-helpers')
+const { lockFixtures } = require('@unlock-protocol/hardhat-helpers')
```

Comment:
Can we just delete this file and import the helper where we import this file? That could be cleaner...